src/actions/start-signature-session.js

```diff
@@ -0,0 +1,25 @@
+const diva = require('diva-irma-js');
+
+/**
+ * Request handler for starting a new disclosure session via POST request
+ * @function requestHandler
+ * @param {object} req Express request object, containing an IRMA content object
+ * @param {object} res Express response object
+ * @returns {undefined}
+ */
+module.exports = function requestHandler(req, res) {
+  const content = req.body.content;
+  if (content) {
+    diva
+      .startSignatureSession(content, null, 'I owe you everything')
```

Comment:
In the next iteration we update the content of the document to be signed to a policy.